vitest.config.ts

```diff
@@ -0,0 +1,18 @@
+import swc from 'unplugin-swc';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+    plugins: [
+        // SWC required to support decorators used in test plugins
+        // See https://github.com/vitest-dev/vitest/issues/708#issuecomment-1118628479
+        // Vite plugin
+        swc.vite({
```

Comment:
Use `swc` in vitest instead of `esbuild` because of some Typescript decorator transpilation thing: https://github.com/nestjs/nest/issues/9228

Didn't go into details to find out, but I saw that Vendure/Michael is also using it this way in @vendure/core.